### DISCUSSION TITLE
Hello Terraform, how to show website

### DISCUSSION THREAD

**Discussion by DriezL** [alkampfergit/personal-blog#4] [Q&A]
Thank you for this excellent blog about an 'Hello world' on azure with Terraform. My question however, if I go to the public IP with my browser, I do not get the nginx site. Do I have to type some more than just the IP? 
Thanks in advance, Dries

**Comment by DriezL** [comment]
I have updated the code from your repository and it works! Thanks. No answer needed.

**Reply by alkampfergit** [reply]
Ok perfect :). Glad for being useful.
